eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-conway
-version:            1.16.0.1
+version:            1.16.1.0
```

Comment:
Hmm, I could have sworn that I bumped this when I updated the `CHANGELOG` last time but apparently I did not. Good catch!